app/views/necesidad/necesidades.html

```diff
@@ -1,10 +1,10 @@
 <style>
-  #listaNec{
-    border: 0;
-    width: 100%;
-    height: 60vh;
-  }
-  </style>
+#listaNec{
```

Comment:
Para que este selector funcione con la L17:
```suggestion
.listaNec{
```